Sources/Cocoa/Components/LabelTextView.swift

```diff
@@ -197,14 +209,47 @@ extension LabelTextView {
     /// it was given when created.
     open override var accessibilityTraits: UIAccessibilityTraits {
         get {
-            guard UIAccessibility.isVoiceOverRunning else {
-                // If the "show numbers" feature is one we want to show the label.
-                return super.accessibilityTraits == .link ? .button : super.accessibilityTraits
-            }
-
             let isTitle = font?.textStyle?.isTitle ?? false
             return isTitle ? .header : super.accessibilityTraits
         }
         set { super.accessibilityTraits = newValue }
     }
 }
+
+/// Class for showing `accessibilityElements` avaliable for Voice Control
+/// and hidding them for VoiceOver.
+private class VoiceControlWrapper: XCView {
+    private let textView: UITextView
```

Comment:
This could cause a retain cycle since the text view also keeps a reference to the VoiceControlWrapper